src/templates/rust-all/.github/workflows/nightly.yml

```diff
@@ -0,0 +1,36 @@
+## DO NOT EDIT!
+# This file was provisioned by Terraform
+# File origin: https://github.com/Arrow-air/tf-github/tree/main/src/templates/rust-all/.github/workflows/rust_ci.yml
+#
+on:
+  schedule:
+    - cron: '0 4 * * *' # run at 4 AM UTC
+
+name: Nightly Build
+
+env:
+  TERM: xterm
+
+jobs:
+  # Will catch broken dependencies
+  build_and_test_debug:
+    name: Build & Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: develop
+      - name: Build
+        run: make build
```

Comment:
Just to be sure, this will also run the docker build. If this is intended, just keep in mind it adds some build time.
If you just want to test the code build, you can use `make rust-build` instead.

Reply:
since `make rust-build` uses a container to build, I think that's enough to test

for example, I encounter errors trying to compile the godot gdnative code through the arrow-rust container because of some libclang dynamic linking issue with the alpine docker, but it compiles fine with `cargo build`

I'll change this to make rust-build